File: C programming/Assignment/Assignment5/type3/factorial.c
#include <stdio.h>

void fact(int num);
void main(){
    int num;
    printf("enter the number:");
    scanf("%d",&num);
    fact(num);
}
void fact(int num){
    int fact =1;
    while(num!=0){
        fact = fact * num;
        num--;
    }
    printf("factorial is %d",fact);
}

File: Java_programming/basic_java/assignment3/factorial.java
package assignment3;

public class factorial {

	public static void main(String[] args) {
		int num=5,fact=1;
//	     printf("enter the number:");
//	     scanf("%d",&num);

	     while(num!=0){
	        fact = fact * num;
	        num--;
	     }

	     System.out.printf("factorial is %d",fact);

	}

}
